src/reports.cpp

```diff
@@ -1031,6 +1031,46 @@ static int attack_info(const reports::context& rc, const attack_type &at, config
 			add_text(res, "\n", "");
 		}
 	}
+
+	//below are version 'abilities' of special_tooltips
+	{
+		//If we have a second unit, do the 2-unit specials_context
+		bool attacking = (u.side() == rc.screen().playing_team().side());
+		auto ctx = (sec_u == nullptr) ? at.specials_context(u.shared_from_this(), hex, attacking) :
+						at.specials_context(u.shared_from_this(), sec_u->shared_from_this(), hex, sec_u->get_location(), attacking, std::move(sec_u_weapon));
+
+		boost::dynamic_bitset<> active;
+		const std::vector<std::pair<t_string, t_string>> &specials = at.abilities_special_tooltips(&active);
```

Comment:
```suggestion
		const std::vector<std::pair<t_string, t_string>>& specials = at.abilities_special_tooltips(&active);
```
We're using the space after `&` convention. (see style guide)

Reply:
changed